.github/ISSUE_TEMPLATE/request_new_purl.yml

```diff
@@ -0,0 +1,73 @@
+name: new PURL
+labels: ["new PURL", "help wanted"] 
+description: Request a new PURL
+title: "[NEW] "
+assignees:
+  - saidfathalla
+body:
+- type: input
+  id: pname
+  attributes:
+    label: Contact person name
+    description: The name of the contact person.
+  validations:
+    required: true
+- type: input
+  id: pemail
+  attributes:
+    label: Contact person email.
+    description: The email of the contact person.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Overview in the README.md
+    description: Could you please give a brief description of what this PID is being redirected to?
+    value: |
+     ### Resource name
+      The resource name goes here.
+     ### Overview
+      A general review or summary of the resource.
```

Comment:
should we here (instead of "resouce") ask for "artefact" according to the discussion reg. ontology/resource/artefact we had earlier?